### ISSUE TITLE
Nathan Burgess- Treasurer

### ISSUE THREAD

**Issue by NathanBurgessDev** [HackSocNotts/nominations#13]
# Name: Nathan Burgess

## Treasurer

# Manifesto

In the words of The Notorious B.I.G

*Mo Money, Mo Problems*

However, I do not think this is the case.

Hi, I'm Nathan! A second year Computer Scientist who believes that HackSoc should remain filthy, filthy rich and I and plan to achieve this through **any means necessary**.

As I'm sure you're all aware, HackNotts happens once a year as a fun event for students all around the UK to come and complete their own projects. But for HackSoc it's all about the three C's = Clout, Connections and Cash.

HackSoc being **the only** society with a free membership (as of writing) is something I proudly stand by. (please remember to vote for this in the AGM once again to avoid the wrath of the Student Union)

This means our only *real* source of income is HackNotts.

However, it is also our only *real* source of expendature. 

As such I believe in dedicating phat stacks towards improving HackNotts to gain more clout. This means making the experience for attendees and sponsors as great as it possibly can. Subsequently retaining sponors and attracting new ones for the future.

Gathering sponors for HackNotts is therefore a prime goal and one I fully intend to be a main driving force behind.

I'm also, *unlike some committee candidates*, an active member in the society having been to many a committee meeting (programming café), a large portion of events / workshops this year been to a couple hackathons and volunteered for HackNotts.

TL:DR: HackNotts important, good for public image, good for cashflow, lot of effort I'm willing to put in,  actually care about the society, have been described as "surprisingly competent".

TL:DR TL:DR:  Mo Money, No Problems - The Notorious N.B
